provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/ProvisioningMojo.java

```diff
@@ -83,5 +111,31 @@ public void execute() throws MojoExecutionException, MojoFailureException {
       // We have something like provisio or presto-plugin
       project.getArtifact().setFile(provisioArchive.file());
     }
+    if (artifacts.size() == 0) {
+      return;
+    }
+    checkDuplicates(artifacts);
+    Model model = project.getOriginalModel().clone();
+    List<Dependency> dependencies = getDependencies(artifacts);
+    mergeDependencies(model, dependencies);
+    try {
+      rewritePom(dependencies, model);
```

Comment:
I think this needs to be called in the `generate-resources` phase. I'll try doing this with the `generateDependencies` mojo, adding `project.setPomFile()` there.